components/stacks-network/src/orchestrator.rs

```diff
@@ -939,12 +940,15 @@ rpcport={bitcoin_node_rpc_port}
 working_dir = "/devnet"
 rpc_bind = "0.0.0.0:{stacks_node_rpc_port}"
 p2p_bind = "0.0.0.0:{stacks_node_p2p_port}"
+data_url = "http://127.0.0.1:{stacks_node_rpc_port}"
+p2p_address = "127.0.0.1:{stacks_node_rpc_port}"
```

Comment:
Do you know if these two new fields are replacing the above `rpc_bind`/`p2p_bind` fields?

Reply:
I'm not sure. I've seen them both in the some other nakamoto test configs. But maybe we could ask the blockchain team for some doc about these changes (not sure it exists though)

Reply:
Config had been briefly reviewed by Aaron, seems to be fine